proto/message.py

```diff
@@ -31,6 +33,13 @@
 from proto.primitives import ProtoType
 
 
+def _has_contribute_to_class(value):
+    # Only call contribute_to_class() if it's bound.
+    return not inspect.isclass(value) and hasattr(value, 'contribute_to_class')
+
+class MapValueMessage:
+    isMapValue: bool = True
```

Comment:
This seems to be unused. Is it part of WIP, or is it cruft?

Reply:
Whoops, thought I deleted that.